### PR TITLE
set password admin

### PR DESCRIPTION
**Pull request by yagonobre**

### CODE REVIEW

File: pkg/client/cmd/user.go [hunk 1]
```diff
@@ -136,4 +146,5 @@ func init() {
 	deleteUserCmd.Flags().String("email", "", "user email [required]")
 
 	RootCmd.AddCommand(setUserPasswordCmd)
+	setUserPasswordCmd.Flags().String("user", "", "user to set the password, if not provide will set the current user password")
```
Comment (aguerra) [inline pkg/client/cmd/user.go]: provide -> provided

File: pkg/server/user/user.go [hunk 2]
```diff
@@ -61,23 +61,32 @@ func (dbu *DatabaseOperations) GetUser(email string) (*database.User, error) {
...
 	if err != nil {
 		return err
 	}
+
 	pass, err := bcrypt.GenerateFromPassword([]byte(newPassword), bcrypt.DefaultCost)
 	if err != nil {
 		return teresa_errors.New(
 			teresa_errors.ErrInternalServerError,
-			errors.Wrap(err, fmt.Sprintf("Generating the password hash to user %s", email)),
+			errors.Wrap(err, fmt.Sprintf("Generating the password hash to user %s", u.Email)),
```
Comment (aguerra) [inline pkg/server/user/user.go]: I guess it's email on errors.Wrap as email may be different from u.Email, right?

File: pkg/server/user/user_test.go [hunk 3]
```diff
@@ -111,28 +111,79 @@ func TestDatabaseOperationsSetPassword(t *testing.T) {
...
+	if err = dbu.SetPassword(user, expectedPassword, ""); err != nil {
 		t.Fatal("error trying to set a new password: ", err)
 	}
 	if _, err = dbu.Login(expectedEmail, expectedPassword, time.Second); err != nil {
 		t.Error("error trying to make login with new password: ", err)
 	}
 }
 
-func TestDatabaseOperationsSetPasswordForInvalidUser(t *testing.T) {
+func TestDatabaseOperationsSetPasswordAdmin(t *testing.T) {
```
Comment (aguerra) [inline pkg/server/user/user_test.go]: This Admin suffix is  a little bit misleading as there as tests with non admin users.

File: pkg/server/user/user.go (hunk 2 above)
Comment (yagonobre) [reply pkg/server/user/user.go]: Yes, I'll fix this, thanks!

File: pkg/server/user/user_test.go (hunk 3 above)
Comment (yagonobre) [reply pkg/server/user/user_test.go]: Yes, this is a mistake

**Comment by yagonobre** [comment]
@aguerra updated

**Pull request merged**